Description: Mark failing tests as "CI only".
 Real fix has been published in https://github.com/brucemiller/LaTeXML/pull/2759
 This patch will be disabled upon next release.
Forwarded: https://github.com/brucemiller/LaTeXML/issues/2628
Author: Hilmar Preuße <hille42@debian.org>
Last-Update: 2025-08-08

--- latexml.orig/t/81_babel.t
+++ latexml/t/81_babel.t
@@ -17,7 +17,9 @@
     greek     => { env=>'CI',
       packages => ['greek.ldf', 'lgrenc.def']
       },
-    french    => ['frenchb.ldf', 'numprint.sty'],
+    french    => { env=>'CI',
+      packages => ['frenchb.ldf', 'numprint.sty']
+      },
     page545   => { env=>'CI',
       packages => ['germanb.ldf', 'frenchb.ldf']
       }
